client/src/js/base/SelectBox.js

```diff
@@ -1,11 +1,13 @@
 import styled from "styled-components";
 import { Box } from "./Box";
+import React from "react";
 
-export const SelectBox = styled(Box)`
+export const StyledSelectBox = styled(Box)`
     border: 1px ${props => (props.active ? props.theme.color.blue : props.theme.color.greyLight)} solid;
     border-radius: ${props => props.theme.borderRadius.sm};
     box-shadow: ${props => (props.active ? props.theme.boxShadow.inset : "none")};
-
+    background: white;
```

Comment:
```suggestion
    background-color: ${props => props.theme.color.white};
```
1. `background-color` is the more specific property here.
2. Pull color from Styled Components theme.

Reply:
Fixed